src/main/scala/com.snowplowanalytics.sauna/apis/Sendgrid.scala

```diff
@@ -177,4 +192,382 @@ object Sendgrid {
       case Left(errors) => Left("Cannot extract custom fields information: " + errors.toString)
     }
   }
+
+  /**
+   * Represents an email recipient - may contain the recipient's name,
+   * must contain their email.
+   */
+  case class SendgridEmailObject(
+    email: String,
+    name: Option[String]
+  )
+
+  implicit val sendgridEmailObjectReads: Reads[SendgridEmailObject] = (
+    (JsPath \ "email").read[String] and
+      (JsPath \ "name").readNullable[String]
+    ) (SendgridEmailObject.apply _)
+  implicit val sendgridEmailObjectWrites: Writes[SendgridEmailObject] = Json.writes[SendgridEmailObject]
```

Comment:
Yes - this particular `Writes` (and the entire chain of `Writes` in this file) is used for https://github.com/snowplow/sauna/blob/d31303b529afa0ac81f5c229fa8c2f31123d2bd6/src/main/scala/com.snowplowanalytics.sauna/apis/Sendgrid.scala#L92.